docs/src/development/faq.md

```diff
@@ -1,177 +0,0 @@
-
-Platform.sh is built on Debian.
-
-## If I choose the Development plan, can I use that plan for production?
-
-No. The Development plan provides all the tools to build your website.
-You can create as many development profiles as you wish for yourself and for your team. 
-However, it doesn't allow for full production-level resources on the production branch
-and doesn't allow you to configure a custom domain name.
-
-Once your project is complete and ready for production, you can choose another plan to go live.
-These plans are available on the [pricing page](https://platform.sh/pricing).
-
-## I am getting weird errors when I push (something with `paramiko`..)
-
-Validate the syntax of your YAML file.
-Don't use tabs.
-If all else fails, contact support.
-
-## Which geographic zones does Platform.sh cover?
-
-Platform.sh works with multiple cloud infrastructure providers,
-including Amazon Web Services, Microsoft Azure, Google Cloud, OVHcloud, and Orange.
-Public Grid regions are available in several parts of the world.
-See a [full list of regions](https://platform.sh/regions/).
-
-Dedicated projects can deploy production to any public AWS or Azure region.
-
-## Why did you choose the .sh extension for your domain?
-
-'sh' is the short version of shell.
-
-According to Wikipedia™, in computing
-a [shell](https://en.wikipedia.org/wiki/Shell_(computing)) is a user interface for access to an operating system's services.
-Generally, operating system shells use either a [command-line interface ](https://en.wikipedia.org/wiki/Command-line_interface) (CLI)
-or [graphical user interface](https://en.wikipedia.org/wiki/Graphical_user_interface) (GUI).
-This is exactly what Platform.sh is about:
-Giving developers tools to build, test, deploy, and run great websites!
-
-".sh" is also the top-level domain for Saint Helena, which looks like a lovely island
-and whose motto is: "Loyal and Unshakeable", which we also strive to be.
-
-## IDE Specific Tips
-
-MAMP pro:
-
-In order for MAMP to work well with the symlinks created by the [Platform.sh CLI](../administration/cli/_index.md),
-add the following to the section under Hosts \> Advanced called "Customized virtual host general settings".
-For more details visit [MAMP Pro documentation page](https://documentation.mamp.info/).
-
-```bash
-<Directory>
-        Options FollowSymLinks
-        AllowOverride All
-</Directory>
-```
-
-{{< note >}}
-
-When you specify your document root,
-MAMP follows the symlink and substitutes the actual build folder path.
-This means that when you rebuild your project locally,
-you need to point the document root to the symlink again so that it refreshes the build path.
-
-{{< /note >}}
-
-## Do you support MFA/2FA/TFA authentication?
-
-Yes. We support multi-factor authentication (MFA) and encourage its use.
-To use it, in the Console, go to **My profile** under your name.
-Then click **Security**, where you can set up an **MFA Application**.
```

Comment:
:lady_beetle: This procedure isn't documented anywhere else. This wasn't a good place for it, but it seems like it should be somewhere?

Reply:
Definitely rushed with this one. Added it to the **Administration**>**Console** section.